hieradata_aws/class/integration/authenticating_proxy_db_admin.yaml

```
@@ -0,0 +1,25 @@
+govuk_env_sync::tasks:
+  "pull_authenticating_proxy_production_daily":
+    ensure: "present"
+    hour: "0"
+    minute: "0"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "_production"
```

Don't we need a full name here? 🤔 
```suggestion
    database: "authenticating_proxy_production"
```